### PR TITLE
Updated project to new spec

### PR DESCRIPTION
**Pull request by brianterry**
Updated the project to new spec.

### CODE REVIEW

File: project/mongodb-atlas-project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "typeName": "MongoDB::Atlas::Project",
-    "description": " Each project has its own Monitoring, Backup and Automations associated with the project. Projects within the same organization share the same billing settings.",
+    "description": "Each project has its own Monitoring, Backup and Automations associated with the project. Projects within the same organization share the same billing settings.",
```
[the remark this answers is not in the document]
Comment (themantissa) [reply project/mongodb-atlas-project.json]: Yikes yes, good catch @shum !  Can you put in an internal docs ticket for the reference link and here we likely want something more like the top part of the page: "Atlas provides a hierarchy based on organizations and projects to facilitate the management of your Atlas clusters.  You can put multiple Projects under an Organization.  See https://docs.atlas.mongodb.com/organizations-projects/ for the benefits of using multiple projects within an organization.

[the remark this answers is not in the document]
Comment (shum) [reply project/mongodb-atlas-project.json]: SGTM! @brianterry don't worry about this in your PR. I'll file a separate PR for this stuff. Thank you for your help on updating the spec!

**Comment by brianterry** [comment]
closing per  @shum's request

**Comment by shum** [comment]
Oh woops, I totally miscommunicated there, sorry. I meant that you didn't have to worry about changing the copy in `project/mongodb-atlas-project.json` and I would deal with that in a separate PR. 

Github won't let me reopen the PR but I think the spec update should definitely be merged. Sorry for the crossed wires!

**Pull request closed without merging**